src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/xlf/MicrosoftNetCoreAnalyzersResources.fr.xlf

```diff
@@ -1568,8 +1568,8 @@
         <note />
       </trans-unit>
       <trans-unit id="PreferIsEmptyOverCountMessage">
-        <source>Prefer 'IsEmpty' over 'Count' to determine whether the object contains or not any items</source>
-        <target state="needs-review-translation">Préférez 'IsEmpty' à 'Count' pour déterminer si l'objet contient ou non des éléments.</target>
+        <source>Prefer 'IsEmpty' over 'Count' to determine whether the object is empty.</source>
+        <target state="needs-review-translation">Préférez 'IsEmpty' à 'Count' pour déterminer si l'objet contient des éléments.</target>
```

Comment:
What about "pour déterminer si l'objet est vide" to match the English?

Reply:
@Evangelink I changed the message as per your suggestion. I also adapted the french version of `MicrosoftNetCoreAnalyzersResources` accordingly.